doc/stdenv/stdenv.chapter.md

```diff
@@ -362,6 +364,42 @@ Unless set to `false`, some build systems with good support for parallel buildin
 
 ### Special variables {#special-variables}
 
+#### `__cleanAttrs` {#var-__cleanAttrs}
+
+By default, `mkDerivation` will expose its arguments in the returned package attribute set. This is unnecessary and leads to some confusion and doubt.
```

Comment:
```suggestion
By default, `mkDerivation` will expose its arguments in the returned package attribute set.
This is often unnecessary and may lead to confusion about what's relevant for consumers.
```

We should soften that a bit, because I don't see evidence for the originally definitive-sounding statement.